.eslintrc.js

```diff
@@ -69,7 +69,6 @@ module.exports = {
     'ember-suave/no-const-outside-module-scope': 'off',
     'ember-suave/require-access-in-comments': 'off',
     'ember/no-get': 'error',
-    'ember/no-get-properties': 'error',
```

Comment:
@iamareebjamal Why this rule was removed ?

Reply:
Read the changelogs of the library